core/src/main/java/com/segment/analytics/kotlin/core/State.kt

```diff
@@ -31,7 +31,7 @@ data class System(
                     Settings.serializer(),
                     storage.read(Storage.Constants.Settings) ?: ""
                 )
-            } catch (ex: Exception) {
+            } catch (ignored: Exception) {
```

Comment:
How do we decide which errors we report and which we ignore?

Reply:
for the ones expect a result we should ignore the exception. for example, parsing int from string with default, if an parse exception raises, we should give the default value, instead of raising exception